schema/artist/carousel.js

```diff
@@ -17,7 +20,7 @@ const ArtistCarouselType = new GraphQLObjectType({
   },
 });
 
-const ArtistCarousel = {
+const ArtistCarousel: GraphQLFieldConfig<ArtistCarouselType, *> = {
```

Comment:
So, what is `GraphQLFieldConfig<ArtistCarouselType, *>`

It's something that we use to declare that the ArtistCarousel is a GraphQL Field, that it's source type ( e.g. the thing that comes in through the `resolve` function ) is of the type ArtistCarouselType. I took the second arg as a star from the default `resolve` [function](https://github.com/graphql/graphql-js/blob/4a832566c9eb5e1633ef0724b74e86039c148539/src/execution/execute.js#L1040) ( as that's where the two params eventually get used.